cmd/config-current.go

```diff
@@ -282,7 +282,8 @@ func validateConfig(s config.Config) error {
 		return err
 	}
 
-	return notify.TestNotificationTargets(s, GlobalServiceDoneCh, NewCustomHTTPTransport())
+	return notify.TestNotificationTargets(s, GlobalServiceDoneCh, NewCustomHTTPTransport(),
+		globalNotificationSys.ConfiguredTargetIDs())
```

Comment:
globalNotificationSys would not have initialized at this moment

Reply:
Of course it would be, the call is only happening when server is running.